### ISSUE TITLE
In rhel8.4 GPU-Driver-Container image why install elfutils-libelf

### ISSUE THREAD

**Issue by lengrongfu** [NVIDIA/gpu-operator#599]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._

_**Important Note:  NVIDIA AI Enterprise customers can get support from NVIDIA Enterprise support. Please open a case [here](https://enterprise-support.nvidia.com/s/create-case)**._

### 1. Quick Debug Information
* OS/Version(e.g. RHEL8.6, Ubuntu22.04):  rhel8.4
* Kernel Version: 4.18.0-305.el8.x86_64
* Container Runtime Type/Version(e.g. Containerd, CRI-O, Docker): Containerd
* K8s Flavor/Version(e.g. K8s, OCP, Rancher, GKE, EKS): K8s
* GPU Operator Version: 1.23.1

### 2. Issue or feature description
_Briefly explain the issue in terms of expected behavior and current behavior._

in the `nvcr.io/nvidia/driver:525.105.17-rhel8.4` image having `elfutils-libelf` lib, but code need install it.

https://github.com/NVIDIA/GPU-Driver-Container/blob/368849ac5e27fa0e4382b4702e786fb9868ecbaf/rhel8/nvidia-driver#L86C28-L86C43

**Comment by shivamerla** [comment]
@lengrongfu looks like only elfutils-libelf is present but not elfutils-devel and we are installing both which are required for build.

**Comment by lengrongfu** [comment]
yes, we need to install `elfutils-devel`, but nvidia-driver script execution `dnf install -q -y elfutils-libelf.$DRIVER_ARCH elfutils-libelf-devel.$DRIVER_ARCH`, will install them at the same time, It will require that the version of elfutils-devel is the same as that of elfutils-libelf, but the version of elfutils-libelf has exceeded the version of elfutils-devel provided by redhat.

In our scenario, we are installing offline and have our own rpm warehouse of rhel8.4 iso, but the `elfutils-libelf-devel` version provided by rhel8.4 is `elfutils-libelf-0.182-3.el8.x86_64.rpm`, but `nvcr.io/nvidia/driver:525.105.17-rhel8.4` The `elfutils-libelf` version provided is `elfutils-libelf-0.187-4.el8.x86_64`. There is a mismatch between the two versions.

**Comment by mmusthafapt** [comment]
Same issue here. Using rhel9.4 images. 
FYI rhel9.4 images have RHEL 9.5 base image.
